app/assets/javascripts/components/page_overlay.es6.jsx

```diff
@@ -0,0 +1,74 @@
+class PageOverlay extends Component {
+
+  static get propTypes() {
+    return {
+      showOverlay: React.PropTypes.bool.isRequired,
+    };
+  }
+
+  static get defaultProps() {
+    return {
+      showOverlay: true,
+    };
+  }
+
+  get styles() {
+    return {
+      overlay: {
+        display: 'flex',
+        flexFlow: 'row',
+        alignItems: 'center',
+        position: 'fixed',
+        zIndex: StyleConstants.planes.nine,
+        width: '100vw',
+        height: '100vh',
+        backgroundColor: 'rgba(0, 0, 0, 0.6)',
+      },
+      editBox: {
```

Comment:
Similarly, we can just call this `card` - this will probably be abstracted into its own component in the future.